maxim/logger/writer.py

```
@@ -155,7 +155,7 @@ def upload_file_data(self, add_attachment_log: CommitLog):
             return
         try:
             if "data" not in attachment:
-                scribe().error(
+                scribe().warning(
                     "[MaximSDK] Data is not set for file attachment. Skipping upload"
```

Ruff (0.11.9)

Trailing comma missing

Add trailing comma

(COM812)